File: .sdk/tm/java/test/ExistsTest.java
package JAVAPACKAGE.sdktest;

import static org.junit.jupiter.api.Assertions.assertNotNull;

import org.junit.jupiter.api.Test;

import JAVAPACKAGE.core.BluefinTecsEcrSDK;

public class ExistsTest {

  @Test
  public void testMode() {
    BluefinTecsEcrSDK testsdk = BluefinTecsEcrSDK.testSDK();
    assertNotNull(testsdk, "expected non-nil SDK");
  }
}

File: .sdk/tm/kotlin/test/ExistsTest.kt
package KOTLINPACKAGE.sdktest

import org.junit.jupiter.api.Assertions.assertNotNull
import org.junit.jupiter.api.Test

import KOTLINPACKAGE.core.BluefinTecsEcrSDK

class ExistsTest {

  @Test
  fun testMode() {
    val testsdk = BluefinTecsEcrSDK.testSDK()
    assertNotNull(testsdk, "expected non-nil SDK")
  }
}
